BitmovinComscoreAnalytics/Classes/ComScoreAnalytics.swift

```diff
@@ -4,6 +4,7 @@ import BitmovinPlayer
 public final class ComScoreAnalytics {
     private static let serialQueue = DispatchQueue(label: "com.bitmovin.player.integrations.comscore.ComScoreAnalytics")
     private static var started: Bool = false
+    private static var configuration: ComScoreConfiguration?
```

Comment:
Storing the whole object is not necessary. We could just store the userConsent instead